### ISSUE TITLE
 [suggestion] Easter event 

### ISSUE THREAD

**Issue by howltrek** [Backdash/MonikaModDev#8042]
It's Easter and Monika appears in a nice bunny suit, celebrating and commenting that she had been working on that surprise for a while. 

![conejita sentada 1](https://user-images.githubusercontent.com/90975566/137631227-49b1fe93-f555-4be9-a64c-465d828eca33.png)

Monika tells the player that she hid x eggs in the room and if the player finds them, she will let him see his suit well. 
When the player finds them, there are some full-body sprites and Monika tells the player that they can follow if they want. 

![proyecto conejita cuerpo completo](https://user-images.githubusercontent.com/90975566/137631300-db2dbc9c-1201-4f40-9e56-6618bdcfa8e7.png)

If the answer is "yes" she  tell the player that this time she will hide them throughout the game. 

![proyecto conejita cuerpo completo2](https://user-images.githubusercontent.com/90975566/137631563-9568cde6-74b7-4c29-a8b7-a50e356e0d57.png)

![proyecto conejita cuerpo completo3](https://user-images.githubusercontent.com/90975566/137631581-d03bcc80-ac71-4534-877a-f51d95001f5b.png)

I can collaborate with art, for the event if it takes place.

these are just sketches, to get the idea across.

I am doing the drawings in such a way that the ears will move with Monika's arms, to give her a more adorable look and I am also making a full body sprite, which could move the legs in the same way, making her poses very dynamic.

![molde conejita 3](https://user-images.githubusercontent.com/90975566/137632354-40bc9d68-daea-4540-9454-e803b5dff50a.png)
![molde conejita 4](https://user-images.githubusercontent.com/90975566/137632358-caf58750-7ad2-428a-86db-9508e6315bf3.png)
![molde conejita 2](https://user-images.githubusercontent.com/90975566/137632360-c335cfae-8058-4671-87ce-b8716398c90d.png)

**Comment by howltrek** [comment]
Please check what I've been doing.

There are still a few tweaks left and I accept suggestions, about necessary changes and collaboration.

If someone helps me to make the .json files in a better way, I will be grateful.

And if they want to take the project of the event, even better.

![screenshot0006](https://user-images.githubusercontent.com/90975566/157196981-bf0fa3d3-7474-4493-940f-dba676e0cbd2.png)
![screenshot0009](https://user-images.githubusercontent.com/90975566/157196986-69c39afc-15ec-43f6-8da5-59ada1544602.png)
![screenshot0015](https://user-images.githubusercontent.com/90975566/157196988-b92c3e96-924d-4eb5-b2f7-7072c4a6a714.png)
![screenshot0016](https://user-images.githubusercontent.com/90975566/157196991-fded94ad-ab64-46bc-9da0-24b1dfb60090.png)

**Comment by Justformas** [comment]
> Bunny outfit is nice. I've assigned @kkrosie123 to assist to make this an actual outfit.
> 
> As for an easter event, I don't think we'll be adding an on-easter event anytime soon. The calculations for an easter date is complicated and not really worth it since it's not as widely celebrated as the other 3 holidays we specifically plan for. However, an easter season is a reasonable range of time we can use to introduce an outfit like this.

Yes the first Sunday after the first full moon of spring would be complicated to implement, and manually adding the date for every year on the calendar would be very tedious. I have seen some games do it, but I think having a range would be better anyway. The possible range for Easter is March 22 to April 25.

I also agree that Easter is not as widely celebrated and is somewhat more religious than Christmas, but please do add a bunny outfit that could be introduced in that date range. That's too much of an opportunity to pass up. Egg hunting and maybe certain Easter-type gifts could be a thing too like in some other games, because that stuff doesn't really have to have religious connotations, but those are other ideas to consider.

**Comment by Justformas** [comment]
You know what? Seeing as Easter has passed this year, why should I wait for when or if they're officially implemented, when I don't really have to? Seeing as you uploaded these outfits for us and they work and all. Thanks for that and by extension the bonus affection.

I supposed I should at least say something about these outfits then. Yes I of course think they're nice. Maybe the introduction/reaction dialogue for these gifts will be more customized than the generic dialogue right now. Also when selecting the outfit for Monika to wear, I see there's some custom dialogue that she says, but sometimes just an "o" is displayed for this dialogue. Probably still unfinished though.
